util/src/Pos/Util/Wlog.hs

```diff
@@ -1,23 +1,289 @@
+{-# LANGUAGE Rank2Types      #-}
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
I do encourage you to rethink that. See my comment below.

Reply:
Done.